index.js

```diff
@@ -1054,9 +1070,47 @@ function config(locales, options = null) {
 	return true;
 }
 
+/**
+ * caching config
+ *
+ * @param {object} options
+ * @param {boolean} options.status
+ * @param {boolean} options.isMemoryStatsEnabled
+ * @param {number|null} options.defaultTtl
+ * @returns {boolean}
+ */
+function caching(options = { status: false, isMemoryStatsEnabled: false, defaultTtl: null }) {
+	if (typeof options.status === 'boolean') {
+		if (options.status) {
+			nopeRedis.SERVICE_START();
+		} else {
+			nopeRedis.SERVICE_KILL_SYNC();
+		}
+	}
+	if (typeof options.isMemoryStatsEnabled === 'boolean') {
+		nopeRedis.config({ isMemoryStatsEnabled: options.isMemoryStatsEnabled, defaultTtl: 1300 });
+	}
+	if (typeof options.defaultTtl === 'number') {
+		nopeRedis.config({ defaultTtl: options.defaultTtl });
+	}
+	return true;
+}
```

Comment:
defaultTtl değerini null değil de 1300 olarak verip, sonrasında typeof kontrollerini tek bir if içerisinde kontrol ettiğimizde hepsinden geçerse tek seferde config düzenlemesi daha doğru olmaz mı? kk-date özel ise bu default ayarlar devam eder bkz:

```suggestion
function caching({ status = false, isMemoryStatsEnabled = false, defaultTtl = 1300 }) {
	if (typeof status === 'boolean') {
		if (status) {
			nopeRedis.SERVICE_START();
		} else {
			nopeRedis.SERVICE_KILL_SYNC();
		}
	}
	if (typeof isMemoryStatsEnabled === 'boolean' && typeof defaultTtl === 'number') {
		nopeRedis.config({
			isMemoryStatsEnabled: isMemoryStatsEnabled,
			defaultTtl: defaultTtl,
		});
	} else {
		nopeRedis.config({ isMemoryStatsEnabled: false, defaultTtl: 1300 });
	}
}
```

Reply:
isMemoryStatsEnabled ve defaultTtl ayarı ayrı olarak ayarlanmalıdır aksi takdirde defaultTtl değiştirmek zorunda olduğunda isMemoryStatsEnabled değeride göndermek zorunda kalır.